README.md

```diff
@@ -1,2 +1,151 @@
 # gpii-glob
-GPII Glob is for....
+
+This library provides a means determining a list of relevant files from a given location based on one or more "globbed"
```

Comment:
Should explain rather more about the origins and purpose of the library - on the one hand, as a drop-in replacement for grunt's native globbing, but to address various performance issues. Explain what these are and how they are addressed.

Reply:
I made a pass at this, please review.

Reply:
Good explanation. Remaining typo "means determining"